### ISSUE TITLE
Issue when importing .svg files with display:none css statements included

### ISSUE THREAD

**Issue by BitSalat** [fontforge/fontforge#4764]
when importing svg files to fontforge I ran into an issue with svg files with display:none statements included
e.g. **< xyz style="display : none" >**

font forge ignoers this and displays elements inside the display:none paragrapg as well so the svg files have 
to be clean in this way to be useable

**Comment by jtanx** [comment]
I don't think we do css parsing so yeah, you're better off cleaning it up before passing it to FF for now.

**Comment by BitSalat** [comment]
Thanx for feedback, indeed that's what I am going to do ...
